app/models/transaction.rb

```diff
@@ -64,6 +71,7 @@ class Transaction < ActiveRecord::Base
   validates_presence_of :payment_gateway
 
   monetize :minimum_commission_cents, with_model_currency: :minimum_commission_currency
+  monetize :unit_price_cents, with_model_currency: :unit_price_currency
```

Comment:
I prefer all monetary values to be stored with separate currency to avoid update on one value to affect others. In this case the purpose is to never modify these amounts after insert but still it feels more clean this way.

Haven't used the money rails gem too much so being explicit helped me understand what actually happens here. Makes the code more readable to me.